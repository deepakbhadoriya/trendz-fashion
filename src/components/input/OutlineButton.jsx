import React from 'react';

import styles from './OutlineButton.module.scss';

const OutlineButton = ({ title = 'button', className }) => {
  return <button className={`${styles.button} ${className}`}>{title}</button>;
};

export default OutlineButton;
