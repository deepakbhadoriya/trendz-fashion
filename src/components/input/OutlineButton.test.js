import React from 'react';

import Enzyme from 'enzyme';
import Adapter from '@wojtekmaj/enzyme-adapter-react-17';
import toJson from 'enzyme-to-json';

import OutlineButton from './OutlineButton';

const { mount } = Enzyme;
Enzyme.configure({ adapter: new Adapter() });

describe('<OutlineButton/>', () => {
  it('render correctly (snapshot)', () => {
    const tree = mount(<OutlineButton />);
    expect(toJson(tree)).toMatchSnapshot();
  });
});
