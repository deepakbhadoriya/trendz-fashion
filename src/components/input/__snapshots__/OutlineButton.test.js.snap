// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<OutlineButton/> render correctly (snapshot) 1`] = `
<OutlineButton>
  <button
    className="button undefined"
  >
    button
  </button>
</OutlineButton>
`;
